src/client/http/transport.rs

```diff
@@ -18,43 +20,42 @@ pub struct HttpTransportClient {
 	/// HTTP client,
 	client: hyper::Client<hyper::client::HttpConnector>,
 	/// Configurable max request body size
-	max_request_body_size: usize,
+	max_request_body_size: u32,
```

Comment:
2 GB max size, I changed because I don't like usize because it's implicit i.e. platform dependent.

Could be `u64` as well but I think 2 GB should be sufficient some browsers limit to 2GB (e.g, Firefox)